README.md

```diff
@@ -625,6 +625,10 @@ public class AwsIamAuthenticationSample {
 }
 ```
 
+### Amazon RDS Blue/Green Deployments
```

Comment:
I think this should be in the Known Issues section. Can you confirm with Matthew.

Reply:
Good call out. I'll check with him.

Reply:
Updated to address this.